package.json

```diff
@@ -1,66 +1,67 @@
 {
-  "name": "react-bubble-ui",
-  "version": "1.1.0",
-  "description": "A highly configurable Bubble UI React.js component, similar to the iconic Apple Watch app layout.",
-  "author": "blakesanie",
+  "version": "0.1.2",
   "license": "MIT",
-  "repository": "blakesanie/React-Bubble-UI",
   "main": "dist/index.js",
-  "module": "dist/index.modern.js",
-  "source": "src/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
   "engines": {
     "node": ">=10"
   },
-  "keywords": [
-    "react",
-    "component",
-    "apple watch",
-    "circle",
-    "bubble",
-    "layout",
-    "watchOS",
-    "apple",
-    "iOS",
-    "parallax"
-  ],
   "scripts": {
-    "build": "microbundle-crl --no-compress --format modern,cjs",
-    "start": "microbundle-crl watch --no-compress --format modern,cjs",
-    "prepare": "run-s build",
-    "test": "run-s test:unit test:lint test:build",
-    "test:build": "run-s build",
-    "test:lint": "eslint .",
-    "test:unit": "cross-env CI=1 react-scripts test --env=jsdom",
-    "test:watch": "react-scripts test --env=jsdom",
-    "predeploy": "cd example && npm install && npm run build",
-    "deploy": "gh-pages -d example/build"
+    "start": "tsdx watch",
```

Comment:
Not sure why, but checking out this PR and installing dependencies with 'npm install --save', then 'npm run start' results in MODULE_NOT_FOUND
<img width="1116" alt="Screen Shot 2022-06-18 at 2 01 57 PM" src="https://user-images.githubusercontent.com/38799390/174452795-cd973980-303a-448f-be15-eff240a7af39.png">

Reply:
> Not sure why, but checking out this PR and installing dependencies with 'npm install --save', then 'npm run start' results in MODULE_NOT_FOUND <img alt="Screen Shot 2022-06-18 at 2 01 57 PM" width="1116" src="https://user-images.githubusercontent.com/38799390/174452795-cd973980-303a-448f-be15-eff240a7af39.png">

Not sure why, it is working fine for me
![image](https://user-images.githubusercontent.com/66078274/174455223-ff6e8c2b-daa5-4a9a-93f2-9e9048c42bca.png)